apps/external-devices/device-descriptors/Atmel_ATmega32.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Atmel Corporation",
+    "deviceName": "ATmega32",
```

Comment:
I'd add the caveat "with custom firmware" since the atmel driver doesn't just work with a bare unflashed avr.